corehq/blobs/tests/test_metadata.py

```diff
@@ -20,14 +18,11 @@ class TestMetaDB(TestCase):
     @classmethod
     def setUpClass(cls):
         super(TestMetaDB, cls).setUpClass()
-        cls.rootdir = mkdtemp(prefix="blobdb")
-        cls.db = fsdb.FilesystemBlobDB(cls.rootdir)
+        cls.db = TemporaryFilesystemBlobDB()
 
     @classmethod
     def tearDownClass(cls):
-        cls.db = None
-        rmtree(cls.rootdir)
```

Comment:
does this happen somewhere else now?

Reply:
Yes, it's done by `TemporaryFilesystemBlobDB.clean_db()` which is called by `close()`

https://github.com/dimagi/commcare-hq/blob/0901a7841bdbe5ba4a5e08b5bf15009e38bda8d2/corehq/blobs/tests/util.py#L95-L98